### ISSUE TITLE
re-starting container deletes all keys and certs

### ISSUE THREAD

**Issue by domi-a** [ljishen/pivpn-docker#4]
i start the container and everything works. client can connect.

stop the container and start it again.
directly after the start the folder /etc/openvpn/easy-rsa is deleted.
while all the keys and certificates are in there.
mounting the folder /etc/openvpn outside the container in a volume does not help.

i don't want to wait 40 mins every start nor copy the client.ovpn file to the client every start of the container

**Comment by ljishen** [comment]
Yes, it is indeed a problem. I have realized that but I don't have time to update the code. Just feel free to submit a pull request if you are willing to help!

**Comment by domi-a** [comment]
i wanted to but i can't build the image.
i cloned the repo. to my rpi 3b. triedd to build the image, but at line/command 14 it gets an error.

it is at "RUN curl -fsSL0 https://install.pivpn.io -...................."
it throws error 255

there is no useful error output
im not shure how to proceed or debug

**Comment by ljishen** [comment]
If you familiar with debugging bash script, you can manually run:

$ INSTALLER=/tmp/install.sh
$ curl -fsSL0 https://install.pivpn.io -o "${INSTALLER}"
$ sed -i 's/debconf-apt-progress --//g' "${INSTALLER}"
$ sed -i '/systemctl start/d' "${INSTALLER}"
$ sed -i '/setStaticIPv4 #/d' "${INSTALLER}"
$ chmod +x "${INSTALLER}"
$ bash -x "${INSTALLER}" --unattended --reconfigure

Then you will see on which line the error occurs.

**Comment by RealHayo** [comment]
I have the same problem.
I used your instruction and  i want to let you know that the line:
$ bash -x "${INSTALLER}" --unattended --reconfigure
will open PIVPN setup screen (as you would know of course)
And the other lines did not give any error when executed.
When going through the whole set up process of the installer, everything works..
